tests/Aspire.Oracle.EntityFrameworkCore.Tests/OracleContainerFixture.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using DotNet.Testcontainers.Builders;
+using Testcontainers.Oracle;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Aspire.Oracle.EntityFrameworkCore.Tests;
+
+public sealed class OracleContainerFixture : IAsyncLifetime
+{
+    private readonly IMessageSink _diagnosticMessageSink;
+
+    public OracleContainer? Container { get; private set; }
+
+    public string GetConnectionString() => Container?.GetConnectionString() ??
+        throw new InvalidOperationException("The test container was not initialized.");
+
+    public OracleContainerFixture(IMessageSink diagnosticMessageSink)
+    {
+        _diagnosticMessageSink = diagnosticMessageSink;
+    }
+
+    public async Task InitializeAsync()
+    {
+        if (RequiresDockerAttribute.IsSupported)
+        {
+            Container = new OracleBuilder()
+                .WithPortBinding(1521, true)
+                .WithHostname("localhost")
+                .WithWaitStrategy(Wait
+                    .ForUnixContainer()
+                    .UntilMessageIsLogged("Pluggable database XEPDB1 opened read write")
```

Comment:
@eerhardt Would be awesome to have this on TestDistributedApplicationBuilder to wait for containers ready state. Will check how this can be done, probably the same way ResourceLoggerForwarderService works.